### DISCUSSION TITLE
Pen Tablet input Support

### DISCUSSION THREAD

**Discussion by dhelgerson** [rustdesk/rustdesk#5644] [Feature Request]
hi, Stumbled upon this project trying to find an open source alternative to parsec warp since it supports pen input. is there any way this project supports is or has any goals of supporting it in the future?

**Comment by AliothCancer** [comment]
I think this  could be a very awesome feature since I could use rnote running on my laptop but writing on my ipad, the latency seems good enough in latency mode even if im using a range extender for my wifi, it would enable to use a lot of linux writing app on a ipad like in my case.  Currently seems that there's no way to make the pencil to write on the screen, you can select app settings but no input on the screen, i can write only with fingers.

**Reply by AaronXenos** [reply]
Android tablet with pencil has same problem, there is nothing done when I slide or click with pencil on the remote screen.
Previously, I thought this was a Flutter adaptation issue for Android, but surprisingly, RustDesk doesn't support iPad either.
It's the most basic functionality to support the pencil just like finger touch.

https://github.com/rustdesk/rustdesk/discussions/10504
https://github.com/rustdesk/rustdesk/discussions/6537

https://www.reddit.com/r/FlutterDev/comments/1092h4w/flutter_tablet_app_that_detects_stylus_the_limits/
